manifests/plugin/abrt.pp

```diff
@@ -33,7 +35,8 @@
 #
 class foreman_proxy::plugin::abrt (
   $enabled                 = $::foreman_proxy::plugin::abrt::params::enabled,
-  $version                 = undef,
```

Comment:
I'm not using it, I just noticed it when I was in the module.  

It looked all alone, being the only param not in params.pp.  I can revert it if you'd prefer it the way it was.